tests/conftest.py

```diff
@@ -202,3 +203,17 @@ def _call(*args, **kwargs):
         return calls
 
     return _capcall
+
+
+@pytest.fixture
+def set_nonexistent_timezone(monkeypatch):
+    """Fixture to temporarily set a non-existent time zone."""
+    def patch():
+        with monkeypatch.context() as mp:
+            mp.setenv('TZ', 'XXX-19:17')  # Set to a non-existent time zone
```

Comment:
Might as well make this 
```suggestion
def set_timezone(monkeypatch):
    """Fixture to temporarily set a system time zone."""
    def patch(time_zone: str):
        with monkeypatch.context() as mp:
            mp.setenv('TZ', time_zone)
```

Reply:
I agree - Easy reviewing twhen we are thinking the same thing, though I was going to make `time_zone` a kwarg with your `19:17` value as default.